rest-backend/src/main/java/com/sap/psr/vulas/backend/repo/BugRepositoryImpl.java

```diff
@@ -3,14 +3,15 @@
 
 import javax.persistence.EntityNotFoundException;
 import javax.persistence.PersistenceException;
-import javax.transaction.Transactional;
+
 import javax.validation.constraints.NotNull;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cache.annotation.CacheEvict;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
```

Comment:
Why did you replace the `Transactional`?

Reply:
I came across the two alternatives while digging in an exception thrown by the UPDATE statement of the timestamps which requires to transactional. Then I read that the latter should be used for spring applications.